iclient.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine
```

Comment:
The binary would be built with the target image, otherwise it won't be compatible with the execution env.

Always use the multi stage builder pattern of docker: 
https://docs.docker.com/develop/develop-images/multistage-build/